src/sbvr-api/uri-parser.ts

```diff
@@ -60,8 +64,8 @@ export interface ODataRequest extends ParsedODataRequest {
 	modifiedFields?: ReturnType<
 		AbstractSQLCompiler.EngineInstance['getModifiedFields']
 	>;
-	affectedIds?: number[];
-	pendingAffectedIds?: Promise<number[]>;
+	affectedIds?: string[];
+	pendingAffectedIds?: Promise<string[]>;
```

Comment:
Why did you change these? Technically they could be strings but by default they'll always be numbers (you have to do some sbvr magic to get pinejs to use non-number id fields)

Reply:
See: https://github.com/balena-io/pinejs/pull/675/files/b5f6e0e7b5428c2642790f78c748a70d68998ea9#r1266631186